deploy/kubernetes/base/controller/controller.yaml

```diff
@@ -32,6 +32,7 @@ spec:
             - "--enable-leader-election"
             - "--leader-election-type=leases"
             - "--leader-election-namespace=$(PDCSI_NAMESPACE)"
+            - "--extra-create-metadata"
```

Comment:
Do we want to set this in staging overlays for now? This will update the "stable" overlay, which may cause stable deployments to fail since the older released drivers can't handle the new parameters.

Reply:
Sure, I'll move this to staging.